src/client/request_scheduler.cpp

```diff
@@ -170,8 +170,8 @@ void RequestScheduler::ProcessAligned(RequestContext* ctx) {
         case OpType::READ:
             ctx->done_->GetInflightRPCToken();
             client_.ReadChunk(ctx->idinfo_, ctx->seq_, ctx->offset_,
-                              ctx->rawlength_, ctx->appliedindex_,
```

Comment:
remove `appliedindex_` from ctx

Reply:
I will remove `appliedindex_` in metacache in next step, curvefs depends on this code now.

Reply:
done